test/e2e/all_in_one_test.go

```diff
@@ -162,7 +162,7 @@ func (suite *AllInOneTestSuite) TestAllInOneWithUIConfig() {
 
 	queryPort := randomPortNumber()
 	ports := []string{queryPort + ":16686"}
-	portForward, closeChan := CreatePortForward(namespace, "all-in-one-with-ui-config", "jaegertracing/all-in-one", ports, fw.KubeConfig)
+	portForward, closeChan := CreatePortForward(namespace, "all-in-one-with-ui-config", "all-in-one", ports, fw.KubeConfig)
```

Comment:
There is no change in this PR related to the container org - why is it no longer required?

If the change is intended to help test against the images in other orgs, I know internally the `all-in-one` image is renamed `jaeger-all-in-one` - so this may cause problems here.

Reply:
CreatePortForward is calling GetPod with the podNamePrefix ("all-in-one-with-ui-config") and that part of the image name ("all-in-one") to find the pod in the current namespace whose name begins with that prefix and which has a container with an image that contains the substring.

https://github.com/jaegertracing/jaeger-operator/blob/master/test/e2e/utils.go#L54-L62

I changed it because when testing with OperatorHub on OpenShift the all-in-one image name is "quay.io/maistra/jaeger-all-in-one-rhel7" so using "all-in-one" will find this, where jaegertracing/all-in-one would not.

I could be more specific, or externalize the name, but I don't think we're likely to have collisions here.

Reply:
Ok thanks for the explanation.